pkg/swaggerdocs/docs.go

```diff
@@ -0,0 +1,3864 @@
+// GENERATED BY THE COMMAND ABOVE; DO NOT EDIT
```

Comment:
@g1eny0ung Could I include this file in git? Because the dependencies should also be included (so that these dependencies will be included in `go.sum`)